gbpservice/neutron/tests/unit/nfp/configurator/test_data/fw_test_data.py

```diff
@@ -199,99 +187,76 @@ def fake_sa_req_list(self):
         """
 
         sa_req_list = [{
-            'service_type': 'generic',
-            'resource': 'interfaces',
-            'method': 'configure_interfaces',
-            'context': {
-                'domain': None
-            },
-            'kwargs': {
-                'kwargs': {
-                    'stitching_mac': None,
-                    'service_vendor': 'vyos',
-                    'provider_interface_position': 2,
+                "agent_info": {
+                    "service_vendor": "vyos",
+                    "service_type": "firewall",
+                    "resource": "interfaces",
+                    "context": {
+                        "requester": "device_orch"
+                    }
+                },
+                "method": "configure_interfaces",
+                "resource_data": {
+                    "stitching_interface_index": 3,
+                    "stitching_mac": "fa:16:3e:da:ca:4d",
+                    "provider_interface_index": 2,
+                    "mgmt_ip": "11.0.0.37",
+                    "stitching_cidr": "192.168.0.0/28",
+                    "provider_mac": "fa:16:3e:d9:4c:33",
+                    "provider_ip": "11.0.1.1",
+                    "provider_cidr": "11.0.1.0/24",
+                    "stitching_ip": "192.168.0.3",
                     'rule_info': {
```

Comment:
This can be removed only when orchestrator removes rule_info from the data
that is being sent. If we remove it here, unit test will fail in schema
validator.

Regards,
Deepak

On Wed, Apr 27, 2016 at 12:45 PM, njagadish <notifications@github.com>
wrote:

> In
> gbpservice/neutron/tests/unit/nfp/configurator/test_data/fw_test_data.py
> <https://github.com/oneconvergence/group-based-policy/pull/251#discussion_r61213615>
> :
>
> > +                    "resource": "interfaces",
> > +                    "context": {
> > +                        "requester": "device_orch"
> > +                    }
> > +                },
> > +                "method": "configure_interfaces",
> > +                "resource_data": {
> > +                    "stitching_interface_index": 3,
> > +                    "stitching_mac": "fa:16:3e:da:ca:4d",
> > +                    "provider_interface_index": 2,
> > +                    "mgmt_ip": "11.0.0.37",
> > +                    "stitching_cidr": "192.168.0.0/28",
> > +                    "provider_mac": "fa:16:3e:d9:4c:33",
> > +                    "provider_ip": "11.0.1.1",
> > +                    "provider_cidr": "11.0.1.0/24",
> > +                    "stitching_ip": "192.168.0.3",
> >                      'rule_info': {
>
> Get rid of rule_info
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/oneconvergence/group-based-policy/pull/251/files/9c68054f9caa237f1086b0c64846ae0c6080839c#r61213615>
>